tests/utils_tests/test_timesince.py

```diff
@@ -23,6 +23,10 @@ def test_equal_datetimes(self):
         """ equal datetimes. """
         # NOTE: \xa0 avoids wrapping between value and unit
         self.assertEqual(timesince(self.t, self.t), '0\xa0minutes')
+        self.assertEqual(
+            timesince(self.t, self.t, time_strings={'minute': '%d mins'}),
+            '0\xa0mins'
+        )
```

Comment:
Can we add a test with a real use case?, e.g. with Czech translations:
```diff
diff --git a/tests/utils_tests/test_timesince.py b/tests/utils_tests/test_timesince.py
index 3e2c5ce56e..3bbcfdf0ba 100644
--- a/tests/utils_tests/test_timesince.py
+++ b/tests/utils_tests/test_timesince.py
@@ -2,7 +2,8 @@ import datetime
 import unittest
 
 from django.test.utils import requires_tz_support
-from django.utils import timezone
+from django.utils import timezone, translation
+from django.utils.translation import npgettext_lazy
 from django.utils.timesince import timesince, timeuntil
 
 
@@ -24,6 +25,16 @@ class TimesinceTests(unittest.TestCase):
         # NOTE: \xa0 avoids wrapping between value and unit
         self.assertEqual(timesince(self.t, self.t), '0\xa0minutes')
 
+    def test_equal_datetimes_humanize_time_strings(self):
+        time_strings = {
+            'minute': npgettext_lazy('naturaltime-future', '%d minute', '%d minutes'),
+        }
+        with translation.override('cs'):
+            self.assertEqual(
+                timesince(self.t, self.t, time_strings=time_strings),
+                '0\xa0minut',
+            )
+
     def test_ignore_microseconds_and_seconds(self):
         """ Microseconds and seconds are ignored. """
         self.assertEqual(timesince(self.t, self.t + self.onemicrosecond), '0\xa0minutes')
```

Reply:
Thanks for the test case.
Done.